### ISSUE TITLE
tasks.startTask with adaptive card does not render in New Teams, works in Classic

### ISSUE THREAD

**Issue by matthiasdilger** [OfficeDev/microsoft-teams-library-js#2208]
**Environment**:
teams-js lib version 2.20.0
New Teams version 24060.2804.2740.4803 
https://teams.microsoft.com/v2/

**To reproduce**:
create app tab with the following code
```
<script
  src="https://res.cdn.office.net/teams-js/2.20.0/js/MicrosoftTeams.min.js"
  integrity="sha384-wBC2cOp5ArjIyhxoK1NtXvJb98SbNojaoD6hXpvxWil+g4O1M2rXNtywEcAoQ8bl"
  crossorigin="anonymous"
></script>
<script>
microsoftTeams.app.initialize();
microsoftTeams.app.getContext().then((context) => { 
const card = {
   "type":"AdaptiveCard",
   "version":"1.2",
   "$schema":"http://adaptivecards.io/schemas/adaptive-card.json",
   "body":[
      {
         "type":"TextBlock",
         "text":"This is a test",
         "wrap":true
      }
    ]
};
const taskInfo = {
          title: 'My title',
          height: 'medium',
          width:'medium',
          card:{
              contentType: "application/vnd.microsoft.card.adaptive",
              content: card,
          },
          fallbackUrl: null,
          completionBotId: 'someId',
      };
      const submitHandler = (result) => {
            if(!result) {
                alert('test');
            }           
        }
        microsoftTeams.tasks.startTask(taskInfo, submitHandler);
});
</script>
```

New Teams opens the task with correct size and title but does not render the card. Works correctly in Classic Teams and Android (did not test iOS).

**Comment by Dinesh-MSFT** [comment]
Hi @matthiasdilger - Thanks for raising the query.
We will look into it and let you know the updates.

**Comment by matthiasdilger** [comment]
Just noticed that the `tasks` namespace is deprecated and not working in New Teams and instead I should use `dialog`. However the equivalent replacement in my case is not supported either and throws error 100: https://learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/teamsjs-support-m365#dialogadaptivecardbot
How am I supposed to load adaptive cards from a tab? Replace them all with HTML?

**Comment by TrevorJoelHarris** [comment]
@matthiasdilger Did you close this because you figured out an answer?

**Comment by matthiasdilger** [comment]
@TrevorJoelHarris I found a way to make `tasks.startTask` work with my adaptive card in new Teams in a personal tab. My card is valid and it gets rendered in a bot context, but did not work when opening it with `tasks.startTask`.
There is a different issue that `tasks.startTask` does not work reliably with an adaptive card in a `configurableTab`, there seems to be a race condition, since reloading the tab makes it work. I will open a different issue, so I closed this one.
